components/library/openssl/openssl-1.0.2/manifests/sample-manifest.p5m

```diff
@@ -2364,6 +2364,8 @@ link path=usr/share/man/man3openssl/SSL_get_servername.3openssl \
 link path=usr/share/man/man3openssl/SSL_get_servername_type.3openssl \
     target=SSL_CTX_set_tlsext_servername_callback.3openssl
 file path=usr/share/man/man3openssl/SSL_get_session.3openssl
+link path=usr/share/man/man3openssl/SSL_get_shared_ciphers.3openssl \
+    target=SSL_get_ciphers.3openssl
```

Comment:
Why is this link missing in openssl-1.0.2.p5m?

Reply:
Because many more (compat) links, which are in `sample-manifest`, are missing from `openssl-1.0.2.p5m`, this one just was added in 1.0.2p.